src/pybind/mgr/orchestrator.py

```diff
@@ -1272,7 +1272,10 @@ def __init__(self,
                  rgw_zone_user=None,  # type: Optional[str]
                  system_access_key=None,  # type: Optional[str]
                  system_secret_key=None,  # type: Optional[str]
-                 count=None  # type: Optional[int]
+                 count=None,  # type: Optional[int]
+                 rgw_frontend_ssl_port=443,  # type: Optional[int]
+                 rgw_frontend_ssl_certificate_path="/etc/ceph/rgw.pem",  # type: Optional[str]
+                 rgw_frontend_ssl_certificate=None   # type: Optional[str]
```

Comment:
Again, :-1: for inventing _any_ defaults here.

Reply:
Yup good point. I'll remove defaults and just make them None.